aws_lambda_powertools/logging/formatter.py

```diff
@@ -72,6 +73,7 @@ def __init__(
         use_datetime_directive: bool = False,
         log_record_order: Optional[List[str]] = None,
         utc: bool = False,
+        use_rfc3339_iso8601: bool = False,
```

Comment:
This is a personal opinion, but I think this variable name makes me a little confused. I know RFC3339 is a profile/subset of the ISO8601 format, but it seems a little confusing to me. Maybe just `use_rfc3339_format`?

But again, it's just a personal opinion as a user and makes no difference to the intended purpose.

Reply:
Makes sense, I'll shorten to `use_rfc3339`. What I meant here is that our new format is compliant with both RFC3339 and ISO8601.

I thought leaving `ISO8601` would give the impression we don't support. But if you are confused, I'm sure others will too - I'll keep `ISO8601` as a docstring but shorten the param.

Reply:
Done!